packages/react-router/src/components/FocusContextProvider/FocusContextProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {useRef, useEffect, ReactNode} from 'react';
+import {FocusContext} from '../../context';
+
+export function FocusContextProvider({children}: {children: ReactNode}) {
+  const focusRef = React.createRef<HTMLElement>();
```

Comment:
Drop the `React.`

Reply:
I think we probably also want this ref to be just `{focus(): void}`, which would let them use other objects that abstract over a DOM focus.

Reply:
Oh great idea!